web-client/src/components/maps.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import '../App.css';
+import { makeStyles } from '@material-ui/core/styles';
+
+
+function GoogleMap(props) {
+    const styles = makeStyles({
+        maps:
+        {
+            padding: '2%',
+            float: 'right',
+            marginTop: '-34%',
+            width: "40%"
+        },
+        map:
+        {
+            display: 'block',
+            width: "100%",
+            height: 480,
+        }
+
+    });
+
+    const style = styles();
+
+    return(
+        <div className={style.maps}>
+            <iframe className={style.map} src={props.path} title="This is a unique title"></iframe>
```

Comment:
as with alt - title should inform about iframe

Reply:
ok I fixed my mistakes